packages/shared/lib/core/profile/actions/active-profile/logout.ts

```diff
@@ -24,10 +24,12 @@ export function logout(clearActiveProfile: boolean = false, _lockStronghold: boo
 
     // (TODO): Figure out why we are using a promise here?
     return new Promise((resolve) => {
-        if (get(isSoftwareProfile)) {
-            _lockStronghold && lockStronghold()
-        } else if (isLedgerProfile(type)) {
-            get(isPollingLedgerDeviceStatus) && stopPollingLedgerNanoStatus()
+        if (_lockStronghold && get(isSoftwareProfile)) {
+            lockStronghold()
+        }
+
+        if (isLedgerProfile(type) && isPollingLedgerDeviceStatus) {
```

Comment:
again: `else if`

Reply:
hmm already fixed this in other branch, looks like there merge removed it. Fixed this now